internal/api/httpsrv/handle_user.go

```diff
@@ -0,0 +1,37 @@
+package httpsrv
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/labstack/echo/v4"
+	"go.infratographer.com/permissions-api/pkg/permissions"
+)
+
+const (
+	actionUserGet = "iam_user_get"
+)
+
+func (h *apiHandler) GetUserByID(ctx context.Context, req GetUserByIDRequestObject) (GetUserByIDResponseObject, error) {
+	// Find the owner the user's issuer is on to check permissions.
+	ownerID, err := h.engine.LookupUserOwnerID(ctx, req.UserID)
+	if err != nil {
+		return nil, echo.NewHTTPError(http.StatusNotFound, err)
+	}
```

Comment:
This could fail for reasons that aren't a not found error, right?

Reply:
Indeed, I meant to replace these with switch statements. Thanks.